### DISCUSSION TITLE
Include git describe

### DISCUSSION THREAD

**Discussion by alex-jesper** [AbhyudayaSharma/react-git-info#25] [Q&A] [answered]
Hi,
The code seems to work well, however I need the output from "git describe --tags", which does not seem to be present.

Is it possible to extend the functionality to include that?

Thanks,
Jesper

**Comment by AbhyudayaSharma** [comment]
Hi @alex-jesper. What information would you want to extract from `git describe --tags`? I think lightweight tags should already be included in `gitInfo.tags`.

**Reply by alex-jesper** [reply]
Well, git describe gives be a nice version like so:
`v0.3.0-2-g6e23bc5`
(semanticVersion-#commitsSinceTag-CurrentHash)
If the current commit is on a given tag the will just be the tag-name (ex v0.3.0)
That is essentially the version of my app.

Looking at what was present in the tags array that was not anything similar to that.

**Reply by AbhyudayaSharma** [reply]
Hmm. I don't think we can extract this information from `git log`. We'll have to add another call to git for this. I will convert this discussion to a feature request.

**Reply by alex-jesper** [reply]
No I agree, thanks :)

If you find the feature useful, I could try to implement it?

I have solved my need via environment variables for now.

**Reply by AbhyudayaSharma** [reply]
> If you find the feature useful, I could try to implement it?

Sure, go ahead. That'll be great!

**Comment by AbhyudayaSharma** [comment, marked as answer]
The feature request is now tracked by #26

**Comment by AmitMPawar** [comment]
Great Package, but wanted the last commit author's name and email if working in a team.
